src/test/java/org/codeforamerica/shiba/output/xml/XmlGeneratorIntegrationTest.java

```diff
@@ -96,7 +96,7 @@ void shouldProduceAValidDocument() throws IOException, SAXException, ParserConfi
                     return Map.entry(pageConfiguration.getName(), new PageData(inputDataMap));
                 })
                 .distinct()
-                .collect(toMap(Map.Entry::getKey, Map.Entry::getValue));
+                .collect(toMap(Map.Entry::getKey, Map.Entry::getValue, (address1, address2) -> address1));
```

Comment:
Added a merge function here so it wouldnt throw the duplicate key exception. I dont think we need this check for generating XMLs. I might be useful to have a test that checks for duplicate keys (with exceptions for things like homeAddress) though